uber/templates/preregistration/preregbase.html

```diff
@@ -37,7 +37,7 @@
                 description: 'A single registration; you can register more before paying.',
                 onClick: function () {
                     $('.group_fields').addClass('hide');
-                    $('input[name="badge_type"]').val('{{ c.ATTENDEE_BADGE }}');
+                    if ($.field('badge_type').val() == '{{ c.PSEUDO_GROUP_BADGE }}') { $('input[name="badge_type"]').val('{{ c.ATTENDEE_BADGE }}'); }
```

Comment:
Previously, we would pre-select the right 'badge type' based on a field called amount_extra. However, now that we actually depend on the `badge_type` field being correct in order to select the right badge type box, I had to change this to only alter the value in certain cases.